utm-domains/utm-domain-commons.yaml

```diff
@@ -765,7 +768,7 @@ definitions:
           Time that the operational volume was freed for use by other
           operations.  Should be populated and stored by the USS.
 
-          This value must satisfy:  actual_time_end > effective_time_begin
+          This value MUST satisfy:  actual_time_end > effective_time_begin
```

Comment:
Let's pull the inequality with its condition onto its own line.  Something like:

`If actual_time is not null, then actual_time_end > effective_time_begin`